recipes/cpprestsdk/all/conandata.yml

```diff
@@ -1,40 +1,40 @@
 sources:
-  "2.10.15":
```

Comment:
I'm sure I've seen the guidance that _conandata.yml_ amd _config.yml_ should be reordered so newest versions are listed first, but right now I can't find it, e.g., in https://github.com/conan-io/conan-center-index/blob/edacf97622e46921af5f1717776adfeb0a129135/docs/adding_packages/conandata_yml_format.md 😕

Reply:
It's in the package template right now

Reply:
Ah, e.g. here:
https://github.com/conan-io/conan-center-index/blob/fc7fdca06847ae16a865d6dc541738fff38a5d59/docs/package_templates/cmake_package/all/conandata.yml#L2